tests/foreman/ui/test_activationkey.py

```diff
@@ -1101,12 +1100,46 @@ def test_associate_host(self):
 
         @Assert: Hosts are successfully associated to Activation key
 
-        @Status: Manual
-
         @BZ: 1078676
 
         """
-        pass
+        key_name = gen_string("alpha", 10)
+        with Session(self.browser) as session:
+            make_activationkey(session, org=self.org_name,
+                               name=key_name, env=ENVIRONMENT)
+            self.assertIsNotNone(self.activationkey.search_key(key_name))
+            # Creating VM
+            with VirtualMachine(distro='rhel66') as vm:
+                # Downloading and Installing rpm
+                result = vm.run(
+                    "wget -nd -r -l1 --no-parent -A "
+                    "'*.noarch.rpm' http://{0}/pub/"
+                    .format(self.server_name)
+                )
+                self.assertEqual(
+                    result.return_code, 0,
+                    "failed to fetch katello-ca rpm: {0}, return code: {1}"
+                    .format(result.stderr, result.return_code)
+                )
+                result = vm.run(
+                    'rpm -i katello-ca-consumer*.noarch.rpm'
+                )
+                self.assertEqual(
+                    result.return_code, 0,
+                    "failed to install katello-ca rpm: {0}, return code: {1}"
+                    .format(result.stderr, result.return_code)
+                )
```

Comment:
I believe that this can be made more succinct, given that we have the last command run and the return code in our test results:

```python
self.assertEqual(result.return-code, 0)
```